docker-compose.yml

```diff
@@ -316,12 +319,20 @@ services:
       - keystore:/keystore
     healthcheck:
       <<: *healthcheck-settings
-      test: wget --no-verbose --tries=1 --spider http://person-restapi:9090/swagger-ui/index.html || exit 1
+      test: wget --no-verbose --tries=1 --spider http://callisto-person-restapi:9090/swagger-ui/index.html || exit 1
     networks:
       - callisto
 
   balance-calculator:
     image: "quay.io/ukhomeofficedigital/callisto-balance-calculator:$BALANCE_CALCULATOR_TAG"
+    environment:
```

Comment:
Was this supposed to be included? It doesn't look like it's part of this change so I wanted to check it was intentional